### ISSUE TITLE
Select-all-button doesn't work on textfield in settings

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#1973]
#### Short description
The textfield in settings doesn't respond to the select-all-button of the copy&paste-smartbar.

![Screenshot_20220726-211756_FlorisBoard Beta.jpg](https://user-images.githubusercontent.com/59611881/181093975-4ee888b6-013b-40d5-9677-a932cbbf9a66.jpg)

#### Environment information
- FlorisBoard Version: 0.4.0-alpha01 (87)
- Install Source: Google PlayStore
- Device: Samsung Galaxy S9
- Android: 10

**Comment by patrickgold** [comment]
Thanks for your bug report!

This is true and the culprit is the text field itself. I hope that the upcoming Jetpack COmpose 1.2 release fixes numerous issues around their text fields.

**Comment by patrickgold** [comment]
I just rechecked this issue and it seems like commit 568dfc973dc2d41d39ca07a4826d3aa2a7020d74, which upgrades Compose to 1.2, fixes this issue. This has been confirmed by testing the text field's select all with FlorisBoard Beta, FlorisBoard Debug and Gboard as the keyboard in the latest master state of the Settings UI.

This may also affect other issues with this text field, but for now I only know that this specific issue is fixed. Yay! Closing thus as completed.
